### PR TITLE
Multiple Thread Link Changes + QOL Changes

### PR DESCRIPTION
**Pull request by Yoshizilla315**

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -550,7 +551,7 @@ exports.Formats = [
 		name: "[Gen 7] Sketchmons",
 		desc: [
 			"Pok&eacute;mon gain access to one Sketched move.",
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587743/\">Sketchmons</a>",
+			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587743/\">Sketchmons Resources</a>",
```
Comment (TheImmortal) [inline config/formats.js]: I think you meant to add the resources thread. Not rename the regular thread to resources.

File: config/formats.js [hunk 2]
```diff
@@ -115,7 +116,7 @@ exports.Formats = [
 		desc: [
 			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587196/\">LC Metagame Discussion</a>",
 			"&bullet; <a href=\"https://www.smogon.com/dex/sm/formats/lc/\">LC Banlist</a>",
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587565/\">LC Viability Ranking</a>",
+			"&bullet; <a href=\"http://www.smogon.com/forums/threads/3609771/\">LC Viability Rankings</a>",
```
Comment (TheImmortal) [inline config/formats.js]: https

File: config/formats.js [hunk 3]
```diff
@@ -90,6 +90,7 @@ exports.Formats = [
 		name: "[Gen 7] NU (beta)",
 		desc: [
 			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3606077/\">NU Metagame Discussion</a>",
+			"&bullet; <a href=\"http://www.smogon.com/forums/threads/3607629/\">NU Viability Rankings</a>",
```
Comment (TheImmortal) [inline config/formats.js]: https

File: config/formats.js (hunk 1 above)
Comment (Yoshizilla315) [inline config/formats.js]: Nah I renamed that on purpose. It has all the resources on it so I thought I'd include it. If you don't want it like that it's fine.

File: config/formats.js (hunk 2 above)
Comment (Yoshizilla315) [inline config/formats.js]: easy fix kk

**Pull request merged**